js/src/hooks/useMCIssues.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
This Hook is used to get the issues with an optional filter for getting only one type of issue